src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,374 @@
+                    throw _host.ExceptDecode(ex, "Tensorflow exception triggered while loading model.");
+#pragma warning restore MSML_NoMessagesForLoadContext
+
+                }
+                return new TFSession(graph);
+            }
+
+            private ITensorValueGetter CreateTensorValueGetter<T>(IRow input, bool isVector, int colIndex, TFShape tfShape)
+            {
+                if (isVector)
+                    return new TensorValueGetterVec<T>(input, colIndex, tfShape);
+                else
+                    return new TensorValueGetter<T>(input, colIndex);
+            }
+
+            private ITensorValueGetter CreateTensorValueGetter(IRow input, TFDataType tfType, bool isVector, int colIndex, TFShape tfShape)
+            {
+                var type = TFTensor.TypeFromTensorType(tfType);
+                _host.AssertValue(type);
+                return Utils.MarshalInvoke(CreateTensorValueGetter<int>, type, input, isVector, colIndex, tfShape);
+            }
+
+            private ITensorValueGetter[] GetTensorValueGetters(IRow input)
+            {
+                var srcTensorGetters = new ITensorValueGetter[InputColIndices.Length];
+                for (int j = 0; j < InputColIndices.Length; j++)
+                {
+                    int colIndex = InputColIndices[j];
+                    srcTensorGetters[j] = CreateTensorValueGetter(input, _tfInputTypes[j], IsVectorInput[j], colIndex, _tfInputShapes[j]);
+                }
+                return srcTensorGetters;
+            }
+
+            private Delegate MakeGetter(IRow input)
+            {
+                var type = TFTensor.TypeFromTensorType(_tfOutputType);
+                _host.Assert(type == OutputColType.ItemType.RawType);
+                return Utils.MarshalInvoke(MakeGetter<int>, type, input, OutputColType);
+            }
+
+            private Delegate MakeGetter<T>(IRow input, ColumnType columnType)
+            {
+                _host.AssertValue(input);
+                _host.Assert(typeof(T) == columnType.ItemType.RawType);
+
+                var srcTensorGetters = GetTensorValueGetters(input);
+
+                ValueGetter<VBuffer<T>> valuegetter = (ref VBuffer<T> dst) =>
+                {
+                    var runner = _session.GetRunner();
+                    for (int i = 0; i < InputColIndices.Length; i++)
+                    {
+                        var inputName = InputColNames[i];
+                        runner.AddInput(inputName, srcTensorGetters[i].GetTensor());
+                    }
+
+                    var tensors = runner.Fetch(OutputColName).Run();
+
+                    Contracts.Assert(tensors.Length > 0);
+
+                    var values = dst.Values;
+                    if (Utils.Size(values) < OutputColType.VectorSize)
+                        values = new T[OutputColType.VectorSize];
+
+                    TensorflowUtils.FetchData<T>(tensors[0].Data, values);
```

Comment:
>FetchData [](start = 36, length = 9)

Currently this won't work for integer types, bool and text (because of the different type system in ML.NET).
I think for now we should remove those types from the Tf2MlNetType() method in TensorflowUtils, and if we find out that these output types are commonly used we should add support for them later.